src/pages/ShopPage.vue

```diff
@@ -10,19 +19,238 @@ useHead({
+      } else {
+        throw new Error('Invalid saved shop format');
+      }
+    } else {
+      throw new Error('Invalid saved shop format');
+    }
+  } catch (error) {
+    const defaultShop = { name: 'Default', items: [] };
+    localStorage.setItem('shops', JSON.stringify([defaultShop]));
+    shop.updateShops([defaultShop]);
+  }
+}
+
+const steps: Step[] = [
+  {
+    target: '#v-step-0',
+    content:
+      'This is the item list. Click on a row to show its description and on the + to add it to the shop to the right.',
+    params: {
+      placement: 'auto'
+    }
+  },
+  {
+    target: '#v-step-1',
+    content:
+      'From this window you can define your preferred settings for the random shop generator.',
+    params: {
+      placement: 'bottom'
+    }
+  },
+  {
+    target: '#v-step-2',
+    content:
+      'Clicking this button will generate a new random shop, based on the generator settings previously described.',
+    params: {
+      placement: 'bottom'
+    }
+  },
+  {
+    target: '.v-step-3',
+    content: 'Here you can sort the columns and narrow your search with the various filters.',
+    params: {
+      placement: 'auto'
+    }
+  },
+  {
+    target: '#v-step-4',
+    content:
+      'This is the shop list, where the items you added or randomly generated will be displayed. You can also increase or decrease the number of each individual item.',
+    params: {
+      placement: 'auto'
+    }
+  },
+  {
+    target: '#v-step-5',
+    content: "This is where the selected item's description will be displayed.",
+    params: {
+      placement: 'auto'
+    }
+  }
+];
+
+const options: VTourOptions = {
+  highlight: true,
+  labels: {
+    buttonSkip: 'Close Help',
+    buttonPrevious: 'Previous',
+    buttonNext: 'Next',
+    buttonStop: 'Finish'
+  }
+};
+
+const startTour = () => {
+  if (!tourActive.value) {
+    tourActive.value = true;
+    const tmpCloakFull: item = {
+      core_item: {
+        id: 15913,
+        name: 'Cloak of Illusions',
+        bulk: 0.1,
+        quantity: 1,
+        base_item: null,
+        category: '',
+        description:
+          '<p>This cloak flows, covering and concealing the wearer\'s body. The cloak allows you to cast @UUID[Compendium.pf2e.spells-srd.Item.Figment] as an occult innate cantrip. Although naturally a dull gray, while invested the cloak picks up colors and patterns from its surroundings, granting a +1 item bonus to Stealth checks.</p>\n<p><strong>Activate—Draw Hood</strong> <span class="action-glyph">2</span> (manipulate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You draw the hood up and gain the effects of @UUID[Compendium.pf2e.spells-srd.Item.Invisibility], with the spell\'s normal duration or until you pull the hood back down, whichever comes first. While you are invisible, your <em>figment</em> innate cantrip gains the subtle trait, concealing the observable effects of your spellcasting.</p>',
```

Comment:
This is a mistake. It contains tags that you do not parse in the FE. I suggest also inputting fake mock data (like item-name, simple description etc) instead of real data, otherwise you are increasing maintenance complexity.

Reply:
Yep, forgot to update it after the backend changes